web/themes/custom/lex/scss/main.scss

```diff
@@ -56,16 +56,17 @@ ol, ul, dl {
 .lex-region-events {
   background-color: color(light-gray);
   display: inline-block;
+  width: 100%;
```

Comment:
needed for tablet size viewports and having background color span full screen width.